src/dev-app/dev-app/dev-app-state.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Direction} from '@angular/cdk/bidi';
+
```

Comment:
I didn't put this in a service, because some of the values are used before the app is bootstrapped.